fastlane_core/lib/fastlane_core/cert_checker.rb

```diff
@@ -111,12 +112,12 @@ def self.list_available_developer_id_installer(in_keychain: nil)
       `#{commands.join(' ')}`
     end
 
-    def self.installed_wwdr_certificates
+    def self.installed_wwdr_certificates(keychain)
```

Comment:
from an API compatibility perspective, I wonder if we should change this to

```
def self.installed_wwdr_certificates(keychain: nil)
   keychain ||= wwdr_keychain
```

Same for other methods.

WDYT?

Reply:
do you mean compatibility with other fastlane code which uses this class or users who might use some of its methods?
the first one should already be sorted as (iirc) I didn't change signatures for used methods
as for the second one, there are no clear public/private separation to me so I assumed those methods are implementation detail as they doesn't look like a public api to me
but happy to use your suggestion if you think it would be the right thing to do

Reply:
@rabbitinspace I meant compatibility with potential external code. 

We unfortunately have a grey way of defining API compatibility in fastlane. I know some users are sometimes using parts of the tools programmatically and anything that isn't made `private` can be used. 

So maybe

```ruby
keychain ||= wwdr_keychain # backwards compatibility
```

would make sense.

I'll ask the team if we can agree on what the clarify what the project public API is.

Reply:
thanks! addressed in 6ad8b71bd